fedbadges/rules.py

```diff
@@ -50,6 +51,11 @@
 except ImportError as e:
     log.warn("Could not import email2fas: %r" % e)
 
+# Match OpenID agent strings, i.e. http://FAS.id.fedoraproject.org
+def openid2fas(openid, **config):
+    if re.match('^https?://[a-z][a-z0-9]+.id\.fedoraproject\.org', openid):
```

Comment:
Missing `\` before `.id`.  Also, should there be a `$` after `\.org`, to prevent this from matching `https://user@id.fedoraproject.org.evil.com`?

Reply:
Thanks for catching the "typo" on the . before id ... I had been debating that final $ but had decided to give evil.com a chance :D